### PR TITLE
update BoM and assembly guide

### PR DESCRIPTION
**Pull request by rcooke-warwick**
update assembly guide and BoM for the relay with the following, based on recent manufacturing run feedback

- update drawing so so that c13/14 match the assembly
- update drawing to correctly represent L/N/E terminals
- update example assembly image so that it has the correct coloured wires
- update BoM to have MRP part numbers 

Change-type: patch
Signed-off-by: Ryan Cooke <ryan@balena.io>

### CODE REVIEW

File: documentation/assembly.md [hunk 1]
```diff
@@ -1,19 +1,23 @@
...
 2. Prepare 6 strands of the unsulated cable, 2 blue, 2 brown, 1 yellow/green
 3. Crimp fast release terminals to both ends of the  yellow/green wires
 4. Crimp fast release terminals to one end of each of the brown and blue wires
-5. Screw the bare wire terminals into the NC and COM terminals of the relay according to the diagram
+5. Crimp blue blade terminals to the other end of each brown and blue wire
+6. Screw the blade terminals into the relay screw terminals
 7. Attach the fast release terminals to the C13 and C14 connectors
-9. Place the C13/14 connectors in the right size holes that are cut into the case
-10. Place the top half of the case on top of the bottom half
+8. Place the C13/14 connectors in the right size holes that are cut into the case
```
Comment (Bucknalla) [inline documentation/assembly.md]: Could you specify which side of the case the connectors should be placed in? Right now I dont think its clear which side they should be inserted into

File: documentation/assembly.md [hunk 2]
```diff
@@ -1,19 +1,23 @@
...
+9.  Place the top half of the case on top of the bottom half
 
 ![Example assembly](/documentation/images/example-assembly.jpg)
 
-> **Note:** The above image didn't use the correct colors as electrical wiring standards vary depending on the country. Thus, it should only be used as a reference. Further builds must use the colors indicated in the textual description above.
-
 > **Note:** If the relay board feels flimsy inside the case you may use M2.5x6mm screws to hold it in place
+
+> **Note:** Ensure that the correct colours are used for line (brown), earth (yellow/green) and neutral (blue)
```
Comment (Bucknalla) [inline documentation/assembly.md]: 'Ensure that the correct colours are used for line (brown), earth (yellow/green) and neutral (blue) following EU/UK IEC convention'

**Review by Bucknalla — CHANGES_REQUESTED** [review]
Couple of minor changes but lgtm

**Comment by rcooke-warwick** [comment]
thanks @Bucknalla done, and also added the wire cutter programs

**Review by Bucknalla — APPROVED** [review]
LGTM

**Pull request merged**